docs/operations/metrics.md

```diff
@@ -402,32 +408,82 @@ The following metric is only available if the `EventReceiverFirehoseMonitor` mod
+|------------------------|--------------------------------------------------------------------------------------|----------|------------|
+| `sys/swap/free`        | Free swap                                                                            ||Varies|
+| `sys/swap/max`         | Max swap                                                                             ||Varies|
+| `sys/swap/pageIn`      | Paged in swap                                                                        ||Varies|
+| `sys/swap/pageOut`     | Paged out swap                                                                       ||Varies|
+| `sys/disk/write/count` | Writes to disk                                                                       |`fsDevName`, `fsDirName`, `fsTypeName`, `fsSysTypeName`, `fsOptions`|Varies|
+| `sys/disk/read/count`  | Reads from disk                                                                      |`fsDevName`, `fsDirName`, `fsTypeName`, `fsSysTypeName`, `fsOptions`|Varies|
+| `sys/disk/write/size`  | Bytes written to disk. One indicator of the amount of paging occurring for segments. |`fsDevName`,`fsDirName`,`fsTypeName`, `fsSysTypeName`, `fsOptions`|Varies|
+| `sys/disk/read/size`   | Bytes read from disk. One indicator of the amount of paging occurring for segments.  |`fsDevName`,`fsDirName`, `fsTypeName`, `fsSysTypeName`, `fsOptions`|Varies|
+| `sys/net/write/size`   | Bytes written to the network                                                         |`netName`, `netAddress`, `netHwaddr`|Varies|
+| `sys/net/read/size`    | Bytes read from the network                                                          |`netName`, `netAddress`, `netHwaddr`|Varies|
+| `sys/fs/used`          | Filesystem bytes used                                                                |`fsDevName`, `fsDirName`, `fsTypeName`, `fsSysTypeName`, `fsOptions`|< max|
+| `sys/fs/max`           | Filesystem bytes max                                                                 |`fsDevName`, `fsDirName`, `fsTypeName`, `fsSysTypeName`, `fsOptions`|Varies|
+| `sys/mem/used`         | Memory used                                                                          ||< max|
+| `sys/mem/max`          | Memory max                                                                           ||Varies|
+| `sys/storage/used`     | Disk space used                                                                      |`fsDirName`|Varies|
+| `sys/cpu`              | CPU used                                                                             |`cpuName`, `cpuTime`|Varies|
+
+## OshiSysMonitor
+
+These metrics are only available if the `OshiSysMonitor` module is included.
+
+| Metric                     | Description                                                                          | Dimensions                           | Normal Value        |
```

Comment:
Please fix the table formatting.

Do not add whitespace to make columns longer. Trim excessive white space between columns.